AUDIO/EnableVehicleExhaustPops.md

```diff
@@ -9,8 +9,11 @@ aliases: ["0x2BE4BC731D039D5A"]
 void ENABLE_VEHICLE_EXHAUST_POPS(Vehicle vehicle, BOOL toggle);
 ```
 
+```
```

Comment:
Please remove the code block from the description if it's not actually any code.